lib/stencil-bundle.js

```diff
@@ -276,8 +276,26 @@ function validateTheme(callback) {
 
         callback(null, true);
     });
+} 
+/**
+ * check for dist files
+ * @param {*}
+ * @returns {Boolean}
+ * 
+ */
+Bundle.prototype.distCheck = function () {
+    const destinationFolder = Path.join(this.themePath,'/assets/dist');
+    if (!Fs.existsSync(destinationFolder)) {
+        return false;
+    } else {
+        Readdir(destinationFolder, function(err,files) {
+            if (err || !Fs.existsSync(Path.join(destinationFolder,'theme-bundle.main.js')) || files.length <= 3) {
```

Comment:
This seems a bit magical to me. Why 3 files? Are there specific files that should always be there?

Reply:
report.html, main and main.map.  i was expecting there to at least be a bundle-chunk file.

Reply:
Ok, maybe this should be a list of expected files and a loop to check for them. That would make the code clearer.

Reply:
So the main issue is that the dist folder itself is not being created.  So instead of doing the stringent checking here, I am going to just check for the dist path.